tests/CMakeLists.txt

```diff
@@ -199,6 +199,11 @@ target_link_libraries(su3_test ${TEST_LIBS})
 quda_checkbuildtest(su3_test QUDA_BUILD_ALL_TESTS)
 install(TARGETS su3_test ${QUDA_EXCLUDE_FROM_INSTALL} DESTINATION ${CMAKE_INSTALL_BINDIR})
 
+add_executable(su3_fermion_test su3_fermion_test.cpp)
```

Comment:
If you're test explicitly requires the Laplace operator, you should guard this around the LAPLACE build definition like [here](https://github.com/lattice/quda/blob/fcd462cc1811f6338d0518771c254918b14b38d3/tests/CMakeLists.txt#L93)